settings.gradle

```diff
@@ -1,2 +1,3 @@
 include ':pinwheel-android'
+include ':app'
```

Comment:
Is this intentional to add this? Was there some reason we didn't have this in here before?

Reply:
It is intentional. There was an issue before with having this on, and using Jitpack builds, but the Jitpack builds seem to have issues anyway, and I plan on deprecating Jitpack support in favor of maven central.